core/src/main/java/com/amplifyframework/predictions/models/LanguageType.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.models;
+
+/**
+ * Enum of different types of language. These are the languages
+ * that are recognized and supported by Amazon Translate Service
+ * and Amazon Comprehend Service.
+ *
+ * @see <a href=https://docs.aws.amazon.com/translate/latest/dg/what-is.html>What is Amazon Translate?</a>
+ * @see <a href=https://docs.aws.amazon.com/comprehend/latest/dg/how-languages.html>Detect the Dominant Language</a>
```

Comment:
Yea I felt the same conflict in my head as I wrote these. But in the end, AWS Amplify IS still an AWS product. While I am always striving to make our use-cases as generic as possible, I don't think we should actively fight ideas just because they originated from us.